community/ibm-memcached-dev/README.md

```diff
@@ -0,0 +1,132 @@
+# Memcached
+
+> [Memcached](https://memcached.org/) is an in-memory key-value store for small chunks of arbitrary data (strings, objects) from results of database calls, API calls, or page rendering.
+
+The original work for this helm chart is present @ [kubernetes Charts](https://github.com/kubernetes/charts/tree/master/stable/memcached) Based on the [memcached](https://github.com/bitnami/charts/tree/master/incubator/memcached) chart from the [Bitnami Charts](https://github.com/bitnami/charts) repository.
+
+```bash
+$ helm install stable/ibm-memcached-dev
+```
+## Resources Required
+The chart deploys pods consuming minimum resources. 
+
+## PodSecurityPolicy Requirements
+This chart requires a PodSecurityPolicy to be bound to the target namespace prior to installation. Choose predefined ibm-anyuid-psp PodSecurityPolicy.
+
+## Introduction
+
+This chart bootstraps a [Memcached](https://hub.docker.com/_/memcached/) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Chart Details
+This chart bootstraps a [Memcached](https://hub.docker.com/_/memcached/) deployment on a [Kubernetes](http://kubernetes.io) cluster
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/ibm-memcached-dev
+```
+
+The command deploys Memcached on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+## Prerequisites
+
+- Kubernetes 1.7+ 
+- Tiller 2.7.2 or later
+
+## Installing the Chart
+
+To install the chart
+
+```shell
+helm install --name <release-name> stable/ibm-memcached-dev
+```
+## Verifying the Chart
+
+See NOTES.txt associated with this chart for verification instructions.
+
+## Uninstalling the Chart
+
+To uninstall/delete the deployment:
+
+```shell
+helm delete --purge <release-name>
+```
+
+## Limitations
+
+## Configuration
+
+The following table lists the configurable parameters of the Memcached chart and their default values.
```

Comment:
We need descriptions of all parameters that are configurable by the user via the values.yaml/ICP GUI.